### PR TITLE
Add a license header #2925

### PR DESCRIPTION
**Pull request by khal0988**
Associated Issue: #2925

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* added license header to js and html files in home directory
* added license header to js and html files in bin/
* added license header to js and html files in flow-typed/
*  added license header to .js files in src/actions
* added license header to files in src/client
* added license header to .js files in src/components
* added license header to files in src/reducers and src/selectors
* added license header to some files in src/test
* added license header to all files in src/utils
* added license header to all files in src/workers
* added license header to all files in src

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### CODE REVIEW

File: src/actions/tests/__snapshots__/ast.spec.js.snap [hunk 1]
```diff
@@ -4,8 +4,8 @@ exports[`ast getOutOfScopeLocations with selected line 1`] = `
 Array [
   Object {
     "end": Object {
-      "column": 16,
-      "line": 1,
+      "column": 63,
```
Comment (codehag) [inline src/actions/tests/__snapshots__/ast.spec.js.snap]: hmm interesting that this changed...

Comment (khal0988) [reply src/actions/tests/__snapshots__/ast.spec.js.snap]: this was auto-updated by $npm test -- -u

**Comment by jasonLaster** [comment]
the license comment should be above `// @flow`

so:

```js
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

// @flow
```

instead of:

```js
// @flow

/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

I think you can write a regex to fix this:

```js
/(\/\*.*\n(.*\*.*)+\n(\/\/ @flow)/, "$2\n$1"
```

File: .travis.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at <http://mozilla.org/MPL/2.0/>.
 language: python
```
Comment (jasonLaster) [inline .travis.yml]: lets only have a license header for `/src` files, this means we can drop it for config files, `/assets`, and `/bin` files

this is easy to do w/ `git reset HEAD~1 bin` ...

File: src/test/mochitest/browser_dbg-editor-gutter.js [hunk 3]
```diff
@@ -1,3 +1,7 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
```
Comment (jasonLaster) [inline src/test/mochitest/browser_dbg-editor-gutter.js]: these tests are already licensed below. We don't need it here

**Pull request closed without merging**